### ISSUE TITLE
Project: Graph TicTacToe

### ISSUE THREAD

**Issue by chingucoding** [microsoft/hack-together#91]
### Project name

Graph TicTacToe

### Description

Graph TicTacToe is a Microsoft Graph based WinUI 3 app that allows users play Tic Tac Toe with each other. The state handling and user handling is using the OneDrive and Outlook Microsoft Graph APIs. Users can invite their contacts to play their game; game files are stored using their OneDrive account and shared with the user they invited.

### Repo URL

https://github.com/chingucoding/GraphTicTacToe

### Team members

chingucoding

**Comment by svrooij** [comment]
Yeah, why not use the OneDrive for storing game content. This is the kind of project perfectly suitable for a HACKathon.

**Comment by waldekmastykarz** [comment]
Awesome @chingucoding! Thank you for sharing your hack with us!

**Comment by waldekmastykarz** [comment]
Hey @chingucoding, would you mind recording a short video walkthrough of your hack? Any format will work. It's not required for the hackathon but it would help us showcase your work both internally at Microsoft and publicly in the community. 😊

**Comment by chingucoding** [comment]
Hi @waldekmastykarz sure thing! Created this short video: https://1drv.ms/v/s!AjxxAPYAosMrhKF4_jSL-id2fl38GA?e=B3beXX
If it's not ideal let me know and I'll work on it again!

**Comment by waldekmastykarz** [comment]
Perfect! Thank you!
